source/includes/kubernetes/_k8_releases.md

```diff
@@ -346,19 +374,34 @@ curl -X POST \
 ```js
 // Change to the latest version of a chart
 {
-  "upgradeChart":  "stable/aerospike",
-  "upgradeChart":  1
+	"upgradeChart": {
+		"name": "aerospike",
+		"repository": {
+			"url": "https://aerospike.github.io/aerospike-kubernetes-enterprise"
+		}
+	}
 }
 
 // Change to a specific version of a chart
 {
-  "upgradeChart" : "https://kubernetes-charts.storage.googleapis.com/aerospike-0.3.2.tgz"
+  "upgradeChart": {
+		"name": "aerospike",
+    "version": "4.9.0",
+		"repository": {
+			"url": "https://aerospike.github.io/aerospike-kubernetes-enterprise"
+		}
+	}
 }
 
 // Change the values for the latest version
 {
-  "upgradeChart" : "stable/aerospike",
-  "values": "---\n\"replicaCount\": 3\n"
+  "upgradeChart": {
+		"name": "aerospike",
+		"repository": {
+			"url": "https://aerospike.github.io/aerospike-kubernetes-enterprise"
+		}
+	}
+  "upgradeValues": "\"replicaCount\": 3"
```

Comment:
missing a comma just before. I suggest putting the example in 3 code block sections instead

Reply:
Fixed the comma. With regard to code blocks, should stick to one since that's the format we use elsewhere and it's a easier to read imo.

| Single code block | Multiple code blocks | 
| --- | --- |
|  ![single-code-block](https://user-images.githubusercontent.com/8960233/98387651-accb1f00-201f-11eb-8d1e-18665473abf9.jpg) | ![multiple-code-blocks](https://user-images.githubusercontent.com/8960233/98387692-b6548700-201f-11eb-9089-abbbf4656865.jpg) |